src/pages/derivx/_selling-points.tsx

```diff
@@ -1,10 +1,15 @@
-import React from 'react'
+import React, { ReactElement } from 'react'
 import styled from 'styled-components'
 import { Localize } from 'components/localization'
 import { Text } from 'components/elements/typography'
 import { SectionContainer, Container, Flex } from 'components/containers'
 import device from 'themes/device'
 
+type SellingPointsType = {
+    title: string | ReactElement
```

Comment:
Same with above, if we want `title` to be always localised then we should keep it as only `ReactElement`

Reply:
Is this actual for this code? I have no idea how to do it with 
const selling_points: SellingPointsType[] = [
    {
        title: '100+',
        subtitle: <Localize translate_text="tradable assets" />,
    },
    {
        title: '24/7',
        subtitle: <Localize translate_text="trading" />,
    },
    {
        title: <Localize translate_text="Zero" />,
        subtitle: <Localize translate_text="commission" />,
    },
]

Reply:
I just checked the array 😅  it seems like some have localized some dont. If this is the case, then ur code is fine 👍🏼